### PR TITLE
Pyprof 1 fix resnet run

### PR DESCRIPTION
**Pull request by debermudez**

### CODE REVIEW

File: pyprof/parse/kernel.py [hunk 1]
```diff
@@ -24,7 +24,12 @@ def demangle(name):
     """
 	Demangle a C++ string
 	"""
-    return cxxfilt.demangle(name)
+    result = name
+    try:
+        result = cxxfilt.demangle(name)
+    except:
+        pass
+    return result
```
Comment (matthewkotila) [inline pyprof/parse/kernel.py:32]: could you explain:

1. why `cxxfilt.demangle()` seems to fail for some names?
2. why `name` is an acceptable return value in lieu of a failed call to `cxxfilt.demangle()`?

File: pyprof/prof/linear.py [hunk 2]
```diff
@@ -107,7 +107,7 @@ def __init__(self, d):
         if any(x in d.name for x in Linear.gemmKernels):
             self.op_ = "linear"
         else:
-            assert (d.name in Linear.biasKernels)
+            assert any(x in d.name for x in Linear.biasKernels)
```
Comment (matthewkotila) [inline pyprof/prof/linear.py:110]: could you explain the logic behind switching from:

- checking if the `d.name` string is a direct match with any string within `Linear.biasKernels`

to:

- checking if any string within `Linear.biasKernels` is contained within the `d.name` string

File: pyprof/tests/L0/run_pyprof_data/test_pyprof_data.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
```
Comment (matthewkotila) [inline pyprof/tests/L0/run_pyprof_data/test_pyprof_data.py:1]: if this change is unrelated to this PR, can it be removed and saved for a separate PR?

File: pyprof/tests/L0/run_test.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
```
Comment (matthewkotila) [inline pyprof/tests/L0/run_test.py:1]: if this change is unrelated to this PR, can it be removed and saved for a separate PR?

File: pyprof/tests/L0/run_pyprof_nvtx/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+import test_pyprof_nvtx.TestPyProfNvtx as TestPyProfNvtx
```
Comment (matthewkotila) [inline pyprof/tests/L0/run_pyprof_nvtx/__init__.py:1]: if this change is unrelated to this PR, can it be removed and saved for a separate PR?

File: pyprof/tests/L0/run_pyprof_nvtx/test_pyprof_nvtx.py [hunk 6]
```diff
@@ -0,0 +1,573 @@
+#!/usr/bin/env python3
```
Comment (matthewkotila) [inline pyprof/tests/L0/run_pyprof_nvtx/test_pyprof_nvtx.py:1]: if this change is unrelated to this PR, can it be removed and saved for a separate PR?

File: pyprof/tests/L0/run_pyprof_data/test_pyprof_data.py (hunk 3 above)
Comment (dzier) [reply pyprof/tests/L0/run_pyprof_data/test_pyprof_data.py:1]: It is already

File: pyprof/tests/L0/run_pyprof_nvtx/__init__.py (hunk 5 above)
Comment (dzier) [reply pyprof/tests/L0/run_pyprof_nvtx/__init__.py:1]: It is already

File: pyprof/tests/L0/run_pyprof_nvtx/test_pyprof_nvtx.py (hunk 6 above)
Comment (debermudez) [reply pyprof/tests/L0/run_pyprof_nvtx/test_pyprof_nvtx.py]: This change was due to the rebase.  It should be gone now.

File: pyprof/tests/L0/run_pyprof_nvtx/__init__.py (hunk 5 above)
Comment (debermudez) [reply pyprof/tests/L0/run_pyprof_nvtx/__init__.py]: This change was due to the rebase.  It should be gone now.

File: pyprof/parse/kernel.py (hunk 1 above)
Comment (debermudez) [reply pyprof/parse/kernel.py:32]: Demangle is failing because the name is not properly formatted. I filed a bug with the cutlass team to investigate since it is from their library.
Return the mangled name is just a patch right now. Once the cutlass team gets back to me with the results of the bug investigation, I will update this code.

File: pyprof/prof/linear.py (hunk 2 above)
Comment (debermudez) [reply pyprof/prof/linear.py:110]: Previously, this code was looking to see if the kernal name was included in Linear.biasKernels. That treats the list of bias kernels as exhaustive. Really, what we want to see is if the kernel name includes the substrings from the bias kernel list. 
This approach is further supported because it is the approach that is taken a few lines above for the gemm kernels.

Comment (matthewkotila) [reply pyprof/prof/linear.py:110]: so `d.name` can be longer than the strings in `Linear.biasKernels`? and this condition was observed when we started testing pyprof with different models than the original code must have worked with?

Comment (debermudez) [reply pyprof/prof/linear.py:110]: Yes `d.name` can be longer than the strings in `Linear.biasKernels`.   
I am not sure what level of testing was run against pyprof before we inherited it. 
We are just starting to get pyprof under test so this is all new behavior that we are trying to capture.

File: pyprof/parse/kernel.py [hunk 7]
```diff
@@ -24,7 +24,12 @@ def demangle(name):
     """
 	Demangle a C++ string
 	"""
-    return cxxfilt.demangle(name)
+    result = name
```
Comment (matthewkotila) [inline pyprof/parse/kernel.py:27]: test review
